shopinvader_api_sale_loyalty/routers/cart.py

```diff
@@ -14,11 +14,12 @@
     authenticated_partner_env,
 )
 from odoo.addons.sale.models.sale_order import SaleOrder
-from odoo.addons.shopinvader_api_cart.routers import cart_router
 from odoo.addons.shopinvader_api_cart.schemas import CartTransaction
 
 from ..schemas import LoyaltyCardInput, LoyaltyRewardInput, Sale
 
+cart_router = APIRouter(tags=["carts"])
```

Comment:
@sbidoul @qgroulard I should may be use another name for the router... ? `cart_loyalty_router` ? to avoid name collision etc...